sirepo/simulation_db.py

```diff
@@ -483,13 +457,6 @@ def prepare_simulation(data, run_dir=None):
         list, py.path: pkcli command, simulation directory
     """
     import sirepo.sim_data
-    if run_dir is None:
```

Comment:
is this called anywhere without a run_dir?

Reply:
No. I've made the argument not optional. https://github.com/radiasoft/sirepo/pull/2532/commits/e8d7a943f378eb6fdb6a9315cea4522022b14197